MarkSkalskyi/README.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with [HDRezka](https://rezka.ag)
+
+HDRezka is a website that allows you to watch your favorite movies and series. If you need a guide for signing in, you're in the right place.
+
+**Movies? HDRezka!**
+
+## Step 1: Registration
+
+1. Firstly, visit the HDRezka website at [HDRezka](https://rezka.ag).
+   This is how it looks like:
+   ![Screenshot of the HDRezka website](https://ibb.co/JxMxSZ1)
+
+2. To register, click the button labeled **"Регистрация"** (Registration).
+   ![Register Button](https://ibb.co/ScJ26s2)
+
+3. A registration window will pop up. Enter your *Login*, *E-Mail*, and *Password*.
```

Comment:
Use present tense instead of future. For example, A registration window appears.

Reply:
Use Email not e-mail.